backend/infrahub/message_bus/operations/requests/generator.py

```diff
@@ -10,36 +10,38 @@
 from infrahub.core.constants import GeneratorInstanceStatus
```

Comment:
This file as well as `backend/infrahub/message_bus/messages/request_generator_run.py` should be moved at a different place, I am not sure where.